bitnami/common/templates/_propertiesvalues.tpl

```diff
@@ -0,0 +1,5 @@
+{{- define "common.properties.render" -}}
```

Comment:
Please add this function inside `kafka/_helpers.tpl` instead.

This can not be added inside common due to several reasons:
- This function is specific to Kafka, and I don't see any other chart benefitting from this in a short-term, so there is no point adding it to the common chart at the moment. If another chart could use this functionality in the future we could move it to the common chart.
- Changes to several charts in the same PR are not allowed in our contributing guidelines. To modify the bitnami/common chart, this change should be performed in a separate PR, merged and then update the common chart dependency in Kafka.

Reply:
moved as proposed